Bottom: 1b69bf9cc29bf9627f82a3f919b4da884760a228
Top:    ea701def481e4991f156c530593b727eebd1d6a8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 17:32:39 +0000

Refresh of lesstif-use-coord-type-for

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 7430f85..49009c7 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
